suites/nautilus/ansible/tier-1_deploy_dmcrypt.yaml

```diff
@@ -0,0 +1,71 @@
+#===============================================================================================
+# Tier-level: 1
+# Test-Suite: tier-1_deploy.yaml
+# Test-Case: Deployment and rollover tests
+#
+# Cluster Configuration:
+#    cephci/conf/nautilus/ansible/tier-1_deploy.yaml
+#
+#    7-Node cluster(RHEL-7.9 and above)
+#    3 MONS, 2 MDS, 1 MGR, 3 OSD and 2 RGW service daemon(s)
+#     Node1 - Mon, Mgr, Installer, OSD
+#     Node2 - Mon, Mgr, OSD
+#     Node3 - Mon, OSD
+#     Node4 - Client, Grafana
+#     Node5 - Pool , used for roll-over test case
+#     Node6 - Pool , used for roll-over test case
+#     Node7 - Pool , used for roll-over test case
+#
+# Test Steps:
+#   (1) Install Pre-requisites, and Deploy Ceph using ceph-ansible
+#   (2) Use pool nodes to add/remove MON, OSD, RGW and MDS using roll-over playbook.
+#===============================================================================================
+tests:
+   - test:
+      name: install ceph pre-requisites
+      module: install_prereq.py
+      abort-on-fail: true
+
+   - test:
+      name: ceph ansible
+      polarion-id: CEPH-10961,CEPH-83572954,CEPH-83572951
+      module: test_ansible.py
+      config:
+        is_mixed_lvm_configs: True
+        ansi_config:
+            dmcrypt: true
+            ntp_service_enabled: true
+            ceph_origin: distro
+            ceph_stable_release: nautilus
```

Comment:
This key/value has no impact. Please refer https://docs.ceph.com/projects/ceph-ansible/en/stable-4.0.25/installation/methods.html

Reply:
done